src/FileCollection.ts

```diff
@@ -11,6 +11,10 @@ export class FileCollection {
     return new FileCollection(this.files.filter(callback));
   }
 
+  find(callback: (file: FileCollectionItem) => unknown) {
+    return this.files.find(callback);
```

Comment:
I we would like to stay consistent I think we should return a new FileCollection as well (that is iteratable anyway)

@santimirandarp WDYT ?

Reply:
I we would like to stay consistent I think we should return a new FileCollection as well (that is iteratable anyway)

@santimirandarp WDYT ?

Reply:
I don't really understand the purpose of this method. Is it just to avoid writing `collection.files.find()` ?

Reply:
But find does not exist in FileCollection class, maybe there is some other way that I don't get.

Reply:
Yes, I got confused, thank you.